### PR TITLE
Add button groups to viewer

### PR DESCRIPTION
**Pull request by tancik**
image

### CODE REVIEW

**Comment by brentyi** [comment]
I'm attempting to upstream this into `viser`, will come back to this after that's stable

File: nerfstudio/viewer/app/src/modules/ConfigPanel/ConfigPanel.jsx [hunk 1]
```diff
@@ -51,6 +52,21 @@ function CustomLeva() {
           value: true,
         });
       }, levaConf.settings);
+    } else if (levaConf.type === 'BUTTON_GROUP') {
+      // Add a button group.
+      leafFolder[key] = buttonGroup({
+        label: `Train Speed`,
+        hint: 'Select the training speed, affects viewer render quality, not final render quality',
```
Comment (brentyi) [inline nerfstudio/viewer/app/src/modules/ConfigPanel/ConfigPanel.jsx:59]: @tancik does this `hint` field do anything? It's not in the TypeScript definition, which only has `label` and `opts`: https://github.com/pmndrs/leva/blob/d658bf0b3cb129e11465e7079dcaed82882eb1bc/packages/leva/src/types/public.ts#L59-L64

Comment (brentyi) [reply nerfstudio/viewer/app/src/modules/ConfigPanel/ConfigPanel.jsx:59]: also, any reason to put `Train Speed` in backticks but the hint in single quotes?

Comment (tancik) [reply nerfstudio/viewer/app/src/modules/ConfigPanel/ConfigPanel.jsx:59]: Here? https://github.com/pmndrs/leva/blob/d658bf0b3cb129e11465e7079dcaed82882eb1bc/packages/leva/src/types/public.ts#L147
No reason for different ticks

Comment (brentyi) [reply nerfstudio/viewer/app/src/modules/ConfigPanel/ConfigPanel.jsx:59]: Yeah, but the input of `buttonGroup` is just `ButtonGroupOpts`: https://github.com/pmndrs/leva/blob/d658bf0b3cb129e11465e7079dcaed82882eb1bc/packages/leva/src/helpers/buttonGroup.ts#L9-L11

And `ButtonGroupOpts` only has `label` and `opts` (as linked above)

=> I get an error when I try to assign `hint` to something:
Screenshot 2023-05-15 at 11 19 14 AM

Comment (tancik) [reply nerfstudio/viewer/app/src/modules/ConfigPanel/ConfigPanel.jsx]: aghh, that logic doesn't belong in the js. Fixed now.

**Review by brentyi — APPROVED** [review]

**Review by brentyi — APPROVED** [review]

**Pull request merged**